plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceSerializerModern.java

```diff
@@ -796,6 +775,89 @@ public boolean parseDataSources(
 
     }
 
+    @Nullable
+    private Map<String, Map<String, Map<String, String>>> readSecureCredentials(
+        @NotNull DBPDataSourceConfigurationStorage configurationStorage,
+        @NotNull DataSourceConfigurationManager configurationManager,
+        @Nullable Collection<String> dataSourceIds
+    ) throws DBException {
+        if (configurationManager.isSecure()) {
+            return null;
+        }
+        final String name = "%s%s%s".formatted(
```

Comment:
Why use formatted instead of string concatenation?

Reply:
I though that the concatenation looked ugly, so I replaced it with formatting that's at least more readable IMO.